R/hybrid.R

```diff
@@ -258,31 +313,42 @@ hybrid <- function(
 
   # NOTE: side effect (doParallel has no functionality to clean up)
   doParallel::registerDoParallel(cluster)
+  register_functions_to_cluster(cluster)
 
   check_files(filenames)
   sample_names <- get_sample_name(filenames)
   number_of_samples <- length(sample_names)
-
+  
   message("**** feature extraction ****")
-  extracted <- extract_features(
-    cluster = cluster,
-    filenames = filenames,
-    min_presence = min_pres,
-    min_elution_length = min_run,
-    mz_tol = mz_tol,
-    baseline_correct = baseline_correct,
-    baseline_correct_noise_percentile = baseline_correct_noise_percentile,
-    intensity_weighted = intensity_weighted,
-    min_bandwidth = min_bandwidth,
-    max_bandwidth = max_bandwidth,
-    sd_cut = sd_cut,
-    sigma_ratio_lim = sigma_ratio_lim,
-    shape_model = shape_model,
-    peak_estim_method = peak_estim_method,
-    component_eliminate = component_eliminate,
-    moment_power = moment_power,
-    BIC_factor = BIC_factor
-  )
+  profiles <- snow::parLapply(cluster, filenames, function(filename) {
```

Comment:
snow can be added as an import for the whole script

Reply:
@zargham-ahmad, `parLapply` (and maybe some other `snow`'s functions) are duplicated in the `parallel` package, which is another dependency of apLCMS. That's likely the reason why the package is specified along with the function here. Do you have any clue how R will select the packge if we just move `snow` to imports? (A genuine question, because I've no idea)

Reply:
On the other hand, maybe specifying these functions like `@importFrom snow foo bar` will work as intended.

Reply:
@maximskorik I wasn't actually talking about the explicit calls, what I meant to write here is adding the package in the roxygen statement which we have at the top of each files which mentions the import for that whole script and add them in the namespace if needed (incase other files are not importing that package) to keep things consistent.

Reply:
That's a good point.

Reply:
Done in [5a36833](https://github.com/RECETOX/recetox-aplcms/pull/154/commits/5a36833b0e22bbf440320b814a0d088db1d19a32).